projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -653,6 +653,22 @@ public static function get_red_bubble_alerts() {
 		return $red_bubble_alerts;
 	}
 
+	/**
+	 *  Add relevant red bubble notifications
+	 *
+	 * @param array $red_bubble_slugs - slugs that describe the reasons the red bubble is showing.
+	 * @return array
+	 */
+	public static function add_red_bubble_alerts( array $red_bubble_slugs ) {
+		$welcome_banner_dismissed = \Jetpack_Options::get_option( 'dismissed_welcome_banner', false );
+		if ( self::is_jetpack_user_new() && ! $welcome_banner_dismissed ) {
```

Comment:
I think maybe since this check will always add the`welcome-banner-active` red bubble slug, maybe we can just rely on that to show the banner on the front-end? Instead of passing yet another value to the initial state 🤔

Reply:
Hmmm, interesting idea - I think it's worked out fairly well in this iteration of reafactor.